framework/src/indicators/InternalSideIndicator.C

```diff
@@ -67,7 +67,8 @@ InternalSideIndicator::InternalSideIndicator(const InputParameters & parameters)
 
     _boundary_id(parameters.get<BoundaryID>("_boundary_id")),
 
-    _var(_subproblem.getStandardVariable(_tid, parameters.get<VariableName>("variable"))),
+    _var(static_cast<MooseVariableField<Real> &>(
```

Comment:
The indicator inherits from MooseVariableInterface - which provides methods you should be using to get the var.  It provides a mooseVariableField function that should give you what you need without resorting to casting.

Reply:
Thanks @rwcarlsen  - this was the bit I needed to get it working without having to use the cast!